<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivationSettings</class>
 <widget class="QDialog" name="ActivationSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>671</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeLabelText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Activation Settings </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Offline_Activation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Offline Activation </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Online_Activation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Online Activation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Serial_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="IdFetch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Fetch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Languages/build-MyMosqu-Desktop-Debug/Untitled-1.png);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
